ddcz/context_processors.py

```diff
@@ -5,15 +5,22 @@
 
 def common_variables(request):
     skin = request.session.get("skin", "light")
+
     return {
         "user": request.user,
         "skin": skin,
+        "skin_for_include": skin_for_include(skin),
+        "current_page_url": request.get_full_path(),
         "skin_css_url": staticfiles_storage.url(
-            "skins/%(skin)s/main.css" % {"skin": skin}
+            "skins/%(skin)s/%(skin)s.css" % {"skin": skin}
```

Comment:
Not sure if it increases readability :)

Reply:
I was thinking that maybe in the future we could do some reorganization like having

| skins
| -- light.css
| -- dark.css

and so on. So all of this manipulation would be easier if we had the skin name in the filename. But right now it does not matter if it's this or main.css. I'll let the decision to you. :)

Reply:
I see two options. 

First, the current one, group by skin:

`skins/$skin/the.css`
`skins/$skin/img/logo.png`
`skins/$skin/img/star.png`

Second, group by type:

`css/$skin.css`
`img/$skin-logo.png`
`img/$skin/star.png`

It felt to me like I more often work on skin-by-skin basis rather than resource-by-resource basis, so I opted for the first approach. 

I don't care much either way, but I wouldn't mix them.

Reply:
I don't see the big difference if we use your first option, only with $skin.css. But I'll rename it then.